server/db/migrations/20181124082120-add-recruiter-username-to-comments.js

```diff
@@ -1,17 +1,27 @@
 'use strict'
 
 module.exports = {
-  up: (queryInterface, Sequelize) => {
+  up: async (queryInterface, Sequelize) => {
+    const applicationsComments = 'ApplicationComments'
+    const recruiterUsername = 'recruiterUsername'
+    const tableDefinition
+      = await queryInterface.describeTable(applicationsComments)
+
+    if (tableDefinition[recruiterUsername])
```

Comment:
This seems a bit hacky. Is the database modified somewhere else than migrations? You should know the state of database if migrations are run in order.

Reply:
I don't quite understand why, but every time the migrations are run, an error is produced for the migrations altering existing tables. This happens also when the applications is run for the first time, i.e. the database is created from scratch. It seems that each table is for some reason created with all columns during the first migration, where it is created, even though some of the columns should be introduced in later migrations. Maybe the model file is somehow used for determining the schema, although it should be done based on the migration files alone.

Reply:
I think that is exactly what is happening and is documented in sequelize documentation

Reply:
Finally understood the problem, thanks!